client/my-sites/domains/domain-management/settings/cards/name-servers-card.tsx

```diff
@@ -126,23 +137,34 @@ const NameServersCard = ( {
 				onClick={ handleLearnMoreClick }
 			/>
 		);
-		const notice = translate(
-			'By using custom name servers, you will manage your DNS records with your new provider, not WordPress.com. {{link}}Learn more{{/link}}',
-			{ components: { link } }
-		);
+
+		let notice;
+		if ( isEditingNameServers && hasWpcomNameservers() ) {
```

Comment:
is `isEditingNameServers` required here? It will always be `true` at this point because of the added check above, right?

Reply:
Good point!